packages/p2p/src/components/my-profile/block-user/block-user.scss

```diff
@@ -1,4 +1,9 @@
 .block-user {
+    &__body {
+        padding-top: 3rem;
+        text-align: center;
```

Comment:
Added top padding to increase text distance from search box and align the text to center:
<img width="412" alt="Screenshot 2022-07-20 at 4 45 41 PM" src="https://user-images.githubusercontent.com/103016120/179945080-8460b2d5-d6c0-4308-b33b-026feeefb016.png">